src/dev/flang/fuir/analysis/dfa/Call.java

```diff
@@ -337,6 +352,10 @@ else if (SpecialClazzes.c_unit == _dfa._fuir.getSpecialClazz(_dfa._fuir.clazzRes
 
             result = _instance.readField(_dfa, rf, NO_SITE, this);
           }
```

Comment:
See my comment for `DFA.java:1687`: Would be better if this could just be

```suggestion
    else if (_result != null)
      {
        result = _result;
      }
```

which would require that `DFA.java:1687` would call `returns` with the correct result value for constructors.